CMakeLists.txt

```diff
@@ -212,6 +212,7 @@ if (BUILD_ENCLAVES)
   add_subdirectory(enclave/core)
   add_subdirectory(3rdparty)
   add_subdirectory(libc)
+  add_subdirectory(3rdparty/mbedtls)
```

Comment:
Why are we doing this? For build ordering? It deserves a comment about the dependency at least, but in the CMake idiom, shouldn't this just be an explicit `add_depenencies` on oelibc?

Also, for my understanding, doesn't the link dependency already force the CMake graph to handle this?

Reply:
oh interesting. I though that cmake wants the targets defined before you use them, but it looks like it works fine.

Yeah, this isn't needed. you can also remove the `add_subdirectory(enclave/core)` part and just have `enclave/CMakeLists.txt` do `add_subdirectory`.